src/main.rs

```diff
@@ -65,6 +66,7 @@ const LIST_OPTION: &str = "list";
 const SHELL_OPTION: &str = "shell";
 const TASKS_OPTION: &str = "tasks";
 const FORCE_OPTION: &str = "force";
+const DOCKER_CLI_OPTION: &str = "docker-cli";
```

Comment:
Maybe we can rename `REPO_OPTION` to `DOCKER_REPO_OPTION` (and change the value to `docker-repo`) and then put this new constant just above it? (So they both start with `DOCKER_`.)

Reply:
I think that would increase the consistency. I just wouldn't have made a CLI breaking change without first asking the owner, even if Toast is pre-v1.0.

Reply:
Good catch that this would be a breaking change. I don't have any usage stats to know if this would affect anyone. But I think it's worth it. I didn't realize until now that this flag doesn't match the corresponding field name in the config file. That is a mistake that I'd like to see fixed.

Unfortunately we'd also need to rename it in a couple of files in https://github.com/stepchowfun/toast/tree/main/.github/actions/toast. That might be a little tricky, since that GitHub Action is used by the CI for this PR (there might be a chicken-and-egg problem that requires bypassing the CI to ship). So feel free to ignore this request, and I can make that change later in a separate PR.